subprojects/version-control/src/main/java/org/gradle/vcs/git/internal/GitVersionControlSystem.java

```diff
@@ -55,7 +55,27 @@ public void populate(File workingDir, VersionControlSpec spec) {
         }
     }
 
-    private void cloneRepo(File workingDir, GitVersionControlSpec gitSpec) {
+    public void populate(File workingDir, VersionControlSpec spec) {
+        populate(workingDir, new DefaultVersionRef(), spec);
+    }
+
+    @Override
+    public Set<VersionRef> getAvailableVersions(VersionControlSpec spec) {
+        GitVersionControlSpec gitSpec = cast(spec);
+        Set<VersionRef> versions = Sets.newHashSet();
+        Collection<Ref> refs = Collections.emptyList();
```

Comment:
⭕️ nit, I think this empty list is never used and the versions hash set could be moved lower

Reply:
Done